index.html

```diff
@@ -8,42 +9,220 @@
+                        <li class="tech"><a href="">css </a></li>
+                        <li class="tech"><a href=""> java script </a></li>
+                        <li class="tech"><a href=""> html </a></li>
+                    </ul>
+                </nav>
+                <button>See Projects</button>
+            </div>
+            <!-- card six -->
+        </section>
+        <section class="top-card-one">
+            <h2>My Recent <br> Works</h2>
+            <div class="bd">
+                <img src="images/ImgPlaholder.png" alt="">
+                <h2>Multi-Post Stories <br> Gain+Glory </h2>
+                <nav>
+                    <ul class="technologies">
+                        <li class="tech"><a href="">Ruby on rails</a></li>
+                        <li class="tech"><a href="">css </a></li>
+                        <li class="tech"><a href=""> java script </a></li>
+                        <li class="tech"><a href=""> html </a></li>
+                    </ul>
+                </nav>
+                <button>See Projects</button>
+
+            </div>
+        </section>
+        <!-- ABout me section  -->
+        <section class="top-card-one">
+
+            <div class="l-shape">
+                <h2>About Me</h2>
+                <p class="second-paragraph "> I can help you build a product , feature or website Look through some of
+                    my
+                    work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <button>Get my resume</button>
+        </section>
+
+
+
+        <section class="top-card-one">
+            <div class="fit">
+                <img src="images/icon-languages.svg" alt="" class="ml">
+                <h2>Languages</h2>
+                <nav>
+                    <ul class="col-one">
+                        <li class="col"><a href="">Ruby on rails</a></li>
+                        <li class="col"><a href="">css </a></li>
+                        <li class="col"><a href=""> java script </a></li>
+                        <li class="col"><a href=""> html </a></li>
+                    </ul>
+                </nav>
+
+
+            </div>
+        </section>
+        <!-- FRamework section -->
+        <section class="top-card-one">
+            <div class="fit">
+                <img src="images/icon-frameworks2.svg" alt="" class="ss">
+                <h2>Frameworks</h2>
+                <nav>
+                    <ul class="col-one">
+                        <li class="col"><a href="">Bootstrap</a></li>
+                        <li class="col"><a href="">Ruby on rails</a></li>
+                        <li class="col"><a href=""> RSpec </a></li>
+                        <li class="col"><a href=""> capybara </a></li>
+                        <li class="col"><a href=""> selinium </a></li>
+                    </ul>
+                </nav>
```

Comment:
- [ ] Hi you did a great job but I'll recommend not using the `<nav>` tag here. the nav tag is mostly used in the header for navigation links. I recommend either changing it to a div or even removing it since its only having one child.